app/src/main/java/com/pajato/argus/Database.kt

```diff
@@ -0,0 +1,89 @@
+package com.pajato.argus
+
+import android.content.ContentValues
+import android.content.Context
+import android.database.Cursor
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+import android.provider.BaseColumns
+
+/* Object that defines the table contents */
+object DatabaseEntry : BaseColumns {
+    val TABLE_NAME = "video"
+    val COLUMN_NAME_TITLE = "title"
+    val COLUMN_NAME_NETWORK = "network"
+    @Suppress("ObjectPropertyName")
+    val _ID = BaseColumns._ID
+}
+
+/* Helper object with our shorthand database methods */
+object DatabaseHelper {
```

Comment:
Yup, this does not need to exist just to wrap the functions.

Reply:
Good point, we can get rid of it.